project/Http4sPlugin.scala

```diff
@@ -218,13 +218,13 @@ object Http4sPlugin extends AutoPlugin {
     val boopickle = "1.3.3"
     val caseInsensitive = "0.3.0"
     val cats = "2.2.0"
-    val catsEffect = "3.0-d5a2213"
+    val catsEffect = "3.0.0-M2"
     val catsEffectTesting = "0.4.1"
     val circe = "0.13.0"
     val cryptobits = "1.3"
     val disciplineSpecs2 = "1.1.0"
     val dropwizardMetrics = "4.1.13"
-    val fs2 = "3.0-5158029"
+    val fs2 = "3.0-cd73a32"
```

Comment:
Oh, one question: Does this have anything that -M2 doesn't?

Reply:
yeah, the `tempFile` and `tempDirectory` changes which unblock this PR, and were added today

Reply:
https://github.com/http4s/http4s/pull/3784#discussion_r511556023